### ISSUE TITLE
An error when using restartworkchain

### ISSUE THREAD

**Issue by nim-hrkn** [janssenhenning/aiida-dataframe#7]
Maybe it is because I use still aiida \_\_version\_\_='1.6.9', but an error occurs after all the calculations are successfully finished when using restartworkchain.

A sample script is in sample/restart.py in 
https://github.com/nim-hrkn/aiida-dataframe.

The error occurs when 
```
_OUTPUT_DF = True
```
which outputs FrameData, but no error when
```
_OUTPUT_DF = False
```
which doesn't output FrameData.

The Error messages are
```
...
  File "/usr/lib/python3.7/asyncio/tasks.py", line 249, in __step
    result = coro.send(None)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/processes.py", line 1230, in step_until_terminated
    await self.step()
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/processes.py", line 1216, in step
    self.transition_to(next_state)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/base/state_machine.py", line 335, in transition_to
    self.transition_failed(initial_state_label, label, *sys.exc_info()[1:])
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/base/state_machine.py", line 351, in transition_failed
    raise exception.with_traceback(trace)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/base/state_machine.py", line 320, in transition_to
    self._enter_next_state(new_state)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/base/state_machine.py", line 386, in _enter_next_state
    self._fire_state_event(StateEventHook.ENTERED_STATE, last_state)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/base/state_machine.py", line 299, in _fire_state_event
    callback(self, hook, state)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/plumpy/processes.py", line 326, in <lambda>
    lambda _s, _h, from_state: self.on_entered(cast(Optional[process_states.State], from_state)),
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/aiida/engine/processes/process.py", line 387, in on_entered
    self.update_node_state(self._state)
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/aiida/engine/processes/process.py", line 610, in update_node_state
    self.update_outputs()
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/aiida/engine/processes/process.py", line 635, in update_outputs
    output.store()
  File "/home/max/.virtualenvs/py39/lib/python3.7/site-packages/aiida_dataframe/data/dataframe.py", line 134, in store
    current_hash = self._hash_dataframe(self._df)
AttributeError: 'PandasFrameData' object has no attribute '_df'
```
 I use aiida-lammps for development and still use AiiDA v1.x.

**Comment by nim-hrkn** [comment]
The 'restart' branch in https://github.com/nim-hrkn/aiida-dataframe.

**Comment by janssenhenning** [comment]
Hi @nim-hrkn,

sorry for the long silence. I just published a new release, including a bug fix that could address your problem.
Could you try to install version `0.1.2` of the package and tell me if that fixes your problem?

**Comment by janssenhenning** [comment]
Should be fixed with #12 and #5 (your sample script runs without error on my machine now). I'll close the issue for now.

Feel free to reopen if you have more problems related to this
